src/pages/player/player.less

```diff
@@ -43,4 +43,50 @@
       line-height: 28px;
     }
   }
+
+  .micro-modal {
+    display: none;
+
+    @media (min-width: 960px) {
```

Comment:
That was a decision from design. I guess it takes up too much space in a narrow viewport?